lib/dest/index.js

```diff
@@ -1,17 +1,24 @@
 'use strict';
 
+var assign = require('object-assign');
 var through2 = require('through2');
 var sourcemaps = require('gulp-sourcemaps');
 var duplexify = require('duplexify');
 var sink = require('../sink');
 var prepareWrite = require('../prepare-write');
+var normalizeOption = require('../normalize-option');
 var writeContents = require('./write-contents');
 
 function dest(outFolder, opt) {
   if (!opt) {
     opt = {};
   }
 
+  assign(opt, {
```

Comment:
this is mutating the `opt` object, which is a something we don't want to do.  You can probably just assign to a `var sourcemapsOpt = ` if it isn't used anywhere else.